classes/class-wc-connect-nux.php

```diff
@@ -83,5 +94,168 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 			);
 			return $pointers;
 		}
+
+		public function get_jetpack_install_status() {
+			$status = Jetpack_Install_Status::UNINSTALLED;
+
+			if( 0 === validate_plugin( 'jetpack/jetpack.php' ) ) {
+				$status = Jetpack_Install_Status::INSTALLED;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
+				// installed, but not activated
+				return $status;
+			}
+
+			if ( defined( 'JETPACK_DEV_DEBUG' ) ) {
```

Comment:
Nice catch. I tried to write some unit tests for this, but I didn't get very far when trying to install Jetpack.

On another note, I refactored this function quite a bit. Makes a lot more sense now :D Also fixed the `true` issue.